src/pages/jiyeon/Main/MainJiyeon.js

```diff
@@ -1,7 +1,207 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-function MainJiyeon() {
-  return <h1>hi</h1>;
-}
+import './MainJiyeon.scss';
+import FeedContents from './feed';
+
+const MainJiyeon = () => {
+  const [data, setData] = useState([]);
+  const [feedData, setFeedData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/data.json')
+      .then(response => response.json())
+      .then(result => setData(result));
+
+    fetch('/data/feedData.json')
+      .then(response => response.json())
+      .then(result => setFeedData(result));
+  }, []);
+
+  return (
+    <div className="wrapMain">
+      <div className="navBottomLine">
+        <nav className="navBar">
+          <div className="navLeft">
+            <img src="/images/jiyeon/Nav/instagram.png" alt="home" />
+            <div className="line"></div>
+            <h1 className="mainH1">Westagram</h1>
+          </div>
+          <input className="search" type="text" placeholder="검색" />
+          <ul className="searchedID"></ul>
+          <div className="navRight">
+            <img src="/images/jiyeon/explore.png" alt="explore" />
+            <img src="/images/jiyeon/heart.png" alt="heart" />
+            <img
+              className="profile"
+              src="/images/jiyeon/profile.png"
+              alt="profile"
+            />
+          </div>
+        </nav>
+
+        <div className="profileBox displayNone">
+          <div className="profileBoxTail"></div>
+          <div className="profileBoxInnerBox">
+            <img
+              src="/images/jiyeon/account_circle_FILL0_wght400_GRAD0_opsz48.png"
+              alt="profile"
+            />
+            <span className="profileBoxText">프로필</span>
+          </div>
+          <div className="profileBoxInnerBox">
+            <img
+              src="/images/jiyeon/bookmark_FILL0_wght400_GRAD0_opsz48.png"
+              alt="bookmark"
+            />
+            <span className="profileBoxText">저장됨</span>
+          </div>
+          <div className="profileBoxInnerBox">
+            <img
+              src="/images/jiyeon/settings_FILL0_wght400_GRAD0_opsz48.png"
+              alt="setting"
+            />
+            <span className="profileBoxText">설정</span>
+          </div>
+          <div className="profileBoxInnerBox">
+            <span className="profileBoxText">로그아웃</span>
+          </div>
+        </div>
+      </div>
+
+      <div className="main">
+        <div className="feeds"></div>
+
+        {feedData.map(function (feedData) {
```

Comment:
```suggestion
        {feedData.map(function (feed) {
```
state의 이름과 동일하게 매개변수 이름을 사용하면 혼동될 가능성이 높습니다!
조금 더 특정된 이름으로 사용해주세요!